test/utils/utils_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"errors"
+	"testing"
+)
+
+func TestVerifyClusterInfo(t *testing.T) {
+	testCases := []struct {
+		desc       string
+		content    string
+		numOfNodes int
+		expectErr  error
+	}{
+		{
+			desc:       "Cluster contains a valid currentNodeCount value",
+			content:    "--currentNodeCount: 3\n--",
```

Comment:
This is weird -- what is the actual output text?

Reply:
Updated test cases.